components/cover/basic/src/index.js

```diff
@@ -0,0 +1,90 @@
+import React, {PropTypes} from 'react'
+
+const CoverBasic = (props) => {
+  const buttons = buildButtons(props)
+
+  // Custom css class is attached when component is clickable.
+  const isClickableClass = isClickable(props) ? ' is-clickable' : ''
+
+  return (
+    <div className={'sui-CoverBasic' + isClickableClass} onClick={buildClickHandler(props)}>
+      <div className={(props.gradient ? 'sui-CoverBasic-gradient' : '')}>
+        <img className='sui-CoverBasic-image' src={props.src} />
+      </div>
+      <div className='sui-CoverBasic-buttonList'>
```

Comment:
You could win some render time by checking if there really is some button to show